LMS/Courses/Http/Requests/CreateCourseRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+namespace LMS\Courses\Http\Requests;
+
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateCourseRequest extends FormRequest
+{
+
+    public function authorize()
+    {
+        return true;
+    }
+
+    public function rules()
+    {
+        return [
+            'course_level_id' => 'required|exists:course_levels,id',
+            'title' => 'required|string|max:60',
+            'subtitle' => 'required|string|max:120',
+            'description' => 'required|string',
+            'cover' => 'required|mimes:png,jpeg',
```

Comment:
No arquivo LMS/Courses/Resources/views/newCourse.blade.php. Diz que também aceitará, gif's.

Reply:
Isso ai foi erro de digitação mesmo. Quando for mexer no sistema já tiro esse aviso.